docker/openemr/flex-3.10/php.ini

```diff
@@ -1947,3 +1947,12 @@ ldap.max_links = -1
 ; Local Variables:
 ; tab-width: 4
 ; End:
+[Xdebug]
+zend_extension=/usr/lib/php7/modules/xdebug.so
+xdebug.coverage_enable=0 
+xdebug.remote_enable=1      
+xdebug.remote_connect_back=0
+xdebug.remote_log=/tmp/xdebug.log
+xdebug.remote_autostart=1  
+xdebug.remote_port=9001
+xdebug.idekey = PHPSTORM
```

Comment:
If we were to make this configurable as a docker environment variable(s), which settings above should be configurable for the developer? Once I know this, then can provide guidance on making this a docker environment settings. To get an idea of what I mean, check out how the `REDIS_SERVER` environment setting works.

Reply:
I think Port, and Idkey, and remote_host are three of the main ones.  This is what I added to the compose file.
         ports: 9000:9001
 environment:    
            XDEBUG_CONFIG: remote_host=host.docker.internal
            PHP_IDE_CONFIG: "serverName=localhost"
       
I think this explains it pretty well.
https://serversforhackers.com/s/xdebug-in-docker